example_config_test.go

```diff
@@ -0,0 +1,43 @@
+package jsonnext_test
+
+import (
+	"fmt"
+	"os"
+
+	"foxygo.at/jsonnext"
+	jsonnet "github.com/google/go-jsonnet"
+)
+
+func ExampleConfig_ConfigureVM() {
+	vm := jsonnet.MakeVM()
+
+	cfg := jsonnext.NewConfig()
+	cfg.ExtVars["extvar"] = jsonnext.NewExtStr("hello")
+	cfg.TLAVars["tlavar"] = jsonnext.NewTLACode(`" world"`)
+	cfg.ConfigureVM(vm)
+
+	str, _ := vm.EvaluateSnippet("<literal>", "function(tlavar) std.extVar('extvar') + tlavar")
+	fmt.Println(str)
+	// Output: "hello world"
+}
+
+func ExampleConfig_ConfigureImporter() {
+	vm := jsonnet.MakeVM()
+	i := &jsonnext.Importer{}
+	vm.Importer(i)
+
+	// testdata/importer/hello.txt contains: hello world\n
+	// testdata/importer/mellow.txt contains: mellow world\n
```

Comment:
I was not trying to illustrate "relative file location imports" work. That is expected to work everywhere on unix/linux and should not need demonstrating. The import search path is specific to this library though, so that is demonstrated. Technically, I'm only demonstrating `ConfigureImporter`, but that can best be shown by then using the configured importer.